manager/container.go

```diff
@@ -306,8 +322,8 @@ func newContainerData(containerName string, memoryCache *memory.InMemoryCache, h
 	cont := &containerData{
 		handler:                  handler,
 		memoryCache:              memoryCache,
-		housekeepingInterval:     *HousekeepingInterval,
-		maxHousekeepingInterval:  maxHousekeepingInterval,
+		housekeepingInterval:     jitter(*HousekeepingInterval, JitterFactor),
```

Comment:
So the range of values will be [*housekeepingInterval, 3*HousekeepingInterval)?
That will mean for a 10 second housekeeping, it will be [10, 30), which is not acceptable.
Isn't the goal to spread housekeeping evenly. So wouldn't a factor of `1` be enough?